### DISCUSSION TITLE
New Field of Cigna Files

### DISCUSSION THREAD

**Discussion by yaffsin** [CMSgov/price-transparency-guide#637] [Q&A]
December in_network_files of Cigna has a new field called "derv_tin" that have not been shared in the CMS schemas.
Which I believe it is an addresser to the tin_id in the shared location url.
Should not they manage this with provider_references ids and well divided location urls?
Do you see something similar in other files or is it just a bug in that file?

In_network file:
![image](https://user-images.githubusercontent.com/115798726/206499525-bf749146-6487-4a56-9531-0da8a2960d69.png)

Location Url:
![image](https://user-images.githubusercontent.com/115798726/206500948-3bbb2089-9d2a-4dfb-9871-82e64bff2d94.png)

**Comment by shaselton-usds** [comment]
Without digging into what that attribute is or how it is meant to being used, there is nothing stopping those that produce the file to add additional attributes to the schema. One would hope that documentation accompanies those files.

In theory, you can still run this file through the [schema validator](https://github.com/CMSgov/price-transparency-guide-validator) and get a valid file if all of the required attributes are present along with custom attributes included.
